src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -619,4 +619,12 @@
   <data name="Adds_a_project_reference" xml:space="preserve">
     <value>Adds a project reference.</value>
   </data>
+  <data name="Add_using_for_0" xml:space="preserve">
+    <value>Add 'using {0};'</value>
```

Comment:
It doesn't matter too much, but I am surprised not to see "Simplify" used here. To me, this code action is the same as the existing simplify action, it just doesn't need an existing using directive (which arguably could be a point of confusion anyway)